src/client/components/PhoneNumber/PhoneNumber.tsx

```diff
@@ -74,16 +115,7 @@ export const PhoneNumber: React.FC<Props> = ({ color }) => {
           {phoneNumber?.displayNumber}
         </PhoneLink>
       </InnerWrapper>
-      {isPhoneOpen() ? (
-        <Text color={color}>
-          {textKeys.PHONE_OPEN_TODAY()} {phoneNumber?.opensAt}-
-          {phoneNumber?.closesAt}
-        </Text>
-      ) : (
-        <Text color={color}>
-          {textKeys.PHONE_CLOSED_UNTIL()} {phoneNumber?.opensAt}
-        </Text>
-      )}
+      <PhoneOpeningHours phoneNumber={phoneNumber} color={color} />
```

Comment:
I would prefer to handle the case where `phoneNumber` is not defined right here:

```suggestion
      {phoneNumber ? <PhoneOpeningHours phoneNumber={phoneNumber} color={color} /> : null}
```

Then we don't need to worry about that case inside the component :)